src/Common/parseGlobs.cpp

```diff
@@ -51,7 +51,15 @@ std::string makeRegexpPatternFromGlobs(const std::string & initial_str_with_glob
             ReadBufferFromString buf_range(buffer);
             buf_range >> range_begin >> point >> point >> range_end;
             bool leading_zeros = buffer[0] == '0';
-            size_t num_len = std::to_string(range_end).size();
+
```

Comment:
Totally not related to your PR, but in l. 35, would you like to replace `{}*,` in the comment by `"{", "}", "*" and ","`?